docs/proposals/fluentd-secure-forward.md

```diff
@@ -0,0 +1,35 @@
+# Allow Fluentd to secure forward to other Fluentd/Logstash (over ES-Copy)
```

Comment:
@ewolinetz The only upgrade it seems you could do is to update the base component and its associated plugins for version.  Upgrade with have to be smart enough to determine start and end for fluents only which is kind of what is being requested of the admin command @sosiouxme is designing.